pkg/bff/collaborators.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/trisacrypto/directory/pkg/gds/secrets"
 )
 
+const MaxCollaborators = 50
```

Comment:
Is there a reason we're making this a constant 50 instead of putting it in config?

Reply:
My thought is that it's not something we would need to change very often.